.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 python:
   - 3.6
 install:
-  - python3 setup.py -q install
+  - pip3 install -r requirements/dev.txt
 script:
   - python3 setup.py flake8
-  - python3 setup.py test
```

Comment:
I'd like this to keep working, IMHO it's kind of a standard (no need to know what kind of test software you use)
can you add this in `setup.cfg` ?
```
[aliases]
test=pytest
```

Reply:
switching to `tox` as the "test" is deprecated https://setuptools.readthedocs.io/en/latest/setuptools.html#test-build-package-and-run-a-unittest-suite
See last commit